packages/components/src/components/page-error/page-error.jsx

```diff
@@ -69,11 +69,34 @@ const PageError = ({
                         )}
                     </span>
                 </div>
-                {redirect_label && (
-                    <ButtonLink className='dc-page-error__btn' to={redirect_url} onClick={onClickHandler} size='large'>
-                        <span className='dc-page-error__btn-text dc-btn__text'>{redirect_label}</span>
-                    </ButtonLink>
-                )}
+                <div className='dc-page-error__btn-wrapper'>
+                    {Array.isArray(redirect_url) ? (
```

Comment:
Could we convert `redirect_url` to an array (`redirect_urls`)?
This way we can remove this `? :` and just stick to one data type which improves code.

Reply:
Yeah, we can, but then we will need to convert all existing PageError calls to pass an array instead. Also, `redirect_label` will have to be converted the same way. Do you think it's the best solution in this case?

Reply:
I added separate props for arrays, to remove unnecessary isArray check, but we still need to have this condition to check if array is passed or not. I think we should not stick only to array datatype because throughout the entire system, we usually need to pass only one param, so it doesn't make sense to always pass an array with only one element inside.